src/SignUp.tsx

```diff
@@ -0,0 +1,152 @@
+// import axios from 'axios';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router';
+
+const SignUp = () => {
+	const navigate = useNavigate();
+	const [errors, setErrors] = useState('');
```

Comment:
I think the name errors is misleading. Also can we not give it an empty string initial value?

Reply:
I fixed it